CMakeLists.txt

```diff
@@ -9,10 +9,12 @@ set( CMAKE_EXPORT_COMPILE_COMMANDS ON )
 # C++ 17 standard
 set( CMAKE_CXX_STANDARD 17 )
 set( CMAKE_CXX_STANDARD_REQUIRED ON )
+add_compile_options( -Wall -Wextra )
```

Comment:
I like using -Wpedantic and -Werror as well. This way, the build will fail if there are any compliance issues with the C++ spec. That might help with portability.

How do you feel about including those flags?

Reply:
Good idea, I'll include these as well.